lib/evmone/tracing.hpp

```diff
@@ -0,0 +1,49 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2021 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+#include <evmc/instructions.h>
+#include <memory>
+#include <string_view>
+
+namespace evmone
+{
+using bytes_view = std::basic_string_view<uint8_t>;
```

Comment:
Don't we have this defined somewhere already?

Reply:
Yes, but in some completely different place. Apparently you can alias the same type multiple times.